frontend/src/Pages/AllUser/Login/Login.jsx

```
@@ -0,0 +1,158 @@
+                                Signing In...
+                            </>
+                        ) : (
+                            'Sign in'
+                        )}
+                    </button>
+                </form>
+
+                {/* Sign Up Link - Preserved */}
+                <p className="text-center text-sm text-gray-500 mt-5">
+                    Don't have an account?{" "}
+                    <a href="#" className="text-indigo-600 hover:underline">
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)